src/app/components/experiment-workspace-detail/experiment-workspace-detail.component.html

```diff
@@ -75,7 +75,9 @@ <h3 mat-subheader *ngIf="templateTabLinks['workspace']?.length">Workspace Tabs</
                     {{tab.name}}
                 </mat-panel-title>
                 <mat-panel-description>
-                    {{templateTabDescriptions[tab.resourceKey?.uiTemplateTabId ?? '']}}
+                    <ng-container *ngIf="templateTabObjects[tab.resourceKey?.uiTemplateTabId ?? '']">
+                        {{templateTabObjects[tab.resourceKey?.uiTemplateTabId ?? ''].description}}
+                    </ng-container>
```

Comment:
```suggestion
                    {{templateTabObjects[tab.resourceKey?.uiTemplateTabId ?? '']?.description ?? ''}}
```
Both works and both methods have disadvantages...